tools/clang_tidy/test/clang_tidy_test.dart

```diff
@@ -146,7 +146,7 @@ Future<int> main(List<String> args) async {
       errSink: errBuffer,
     );
     final List<io.File> fileList = await clangTidy.computeChangedFiles();
-    expect(fileList.length, lessThan(300));
+    expect(fileList.length, lessThan(1000));
```

Comment:
This PR only changes 100 files, and not all of those are C++ files, so why does this need to be increased?

Reply:
Hm not sure, it was failing at one point. I'll investigate. Does this check the transitive dep sources too?

Reply:
No, only the sources in flutter/engine.

Reply:
No, it should only be looking at files under `src/flutter` that `git` reports as new or modified on your branch.

Reply:
Put back to 300